javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,21 @@
+/* exported getStudentNames */
+
+/*
+create a function that returns the names of students from objects within arrays
+create a storage container
+access the property name within each object of the array if it has one
+if it does, steal that value and put it in the storage container
+if it doesnt, move on to the next object
+do that x amount of times until there are no more objects with name properties left to check
+return the values of the name property
+*/
+
+function getStudentNames(students) {
+  var studentNames = [];
+  for (var i = 0; i < students.length; i++) {
+    if (typeof students[i].name === 'string') {
```

Comment:
I don't think this conditional is necessary for anything 🤔

Reply:
true, i guess i was thinking that you would want to endure that they entered a valid name ? instead of just a # or something.. but ya, its prob unnecessary . just fixed it 
![Screen Shot 2022-09-01 at 9 51 55 AM](https://user-images.githubusercontent.com/109625982/187969520-2ba148f5-355a-41bb-b8ac-ebe9d3f4e175.png)